packages/spec-test-runner/test/utils/specTestCases.ts

```diff
@@ -1,5 +1,5 @@
 import {join} from "path";
 
-export const SPEC_TEST_REPO_URL = "https://github.com/ChainSafe/eth2.0-spec-tests";
-export const SPEC_TEST_VERSION = "v1.1.0-alpha.3";
+export const SPEC_TEST_REPO_URL = "https://github.com/ethereum/eth2.0-spec-tests";
```

Comment:
could someone add spec test 1.1.0-alpha.5 to our ChainSafe repo @wemeetagain @mpetrunic

Reply:
We don't need this anymore, spec tests are now released in ethereum repo:
https://github.com/ethereum/eth2.0-spec-tests/releases/tag/v1.1.0-alpha.6

We switched to our fork briefly since upstream repo didn't publish lightclient stuff releases